src/core/tests/frontend/paddle/test_models/gen_scripts/generate_sum.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import print_function
+import sys
+import paddle
+import paddle.fluid as fluid
+from save_model import saveModel
+
+paddle.enable_static()
+
+
+def sum(name: str, x, dtype=None):
+    out = paddle.add_n(x)
```

Comment:
Hi 
According to your reference document: https://www.paddlepaddle.org.cn/documentation/docs/zh/1.8/api_cn/layers_cn/sum_cn.html#sum
seems we should use paddle.fluid.layers.sum(x) to simulate paddle's output right?

Reply:
updated